sdcm/sct_config.py

```diff
@@ -2113,6 +2118,14 @@ def _validate_number_of_db_nodes_divides_by_az_number(self):
             assert nodes_num % az_count == 0, \
                 f"Number of db nodes ({nodes_num}) should be divisible by number of availability zones ({az_count})"
 
+    def _validate_placement_group_required_values(self):
+        if self.get("use_placement_group"):
+            az_count = len(self.get('availability_zone').split(',')) if self.get('availability_zone') else 1
+            regions_count = len(self.region_names)
+            assert az_count == 1 and regions_count == 1, \
+                (f"Number of Regions({regions_count}) and AZ({az_count}) should be 1 "
```

Comment:
I think we can still support multi-dc - provisioner provisions region by region

Reply:
we are gonna use placement groups for trying to get stable results, and place the whole test on the same place.
multidc case kind of contradicts that, and I think it's o.k. that we don't  support placement groups for multidc.

i.e. we don't need it in the foreseen future, no need to waste time on implementing/testing it.